test/packer_basic_example_test.go

```diff
@@ -51,6 +52,49 @@ func TestPackerBasicExample(t *testing.T) {
 	assert.True(t, amiIsPublic)
 }
 
+func TestPackerMultipleConcurrentAmis(t *testing.T) {
+	t.Parallel()
+	i := 0
+
+	// Build a map of 3 randomId <-> packer.Options, in 3 random AWS Regions
+	// then build all of these AMIs in parallel and make sure that there are
+	// no errors.
+	var identifierToOptions = map[string]*packer.Options{}
+	for i < 3 {
+		// Pick a random AWS region to test in. This helps ensure your code works in all regions.
+		awsRegion := terratest_aws.GetRandomRegion(t, nil, nil)
```

Comment:
If you happen to get the same region twice, will you get an error about duplicate AMI names? If so, the Packer template may want to take in a variable called `ami_base_name`, set by default to "terratest-packer-example" (the current name). At test time, you can configure `packer.Options` to use a different `random.UniqueId()` for each one as part of the name.

Reply:
fixed